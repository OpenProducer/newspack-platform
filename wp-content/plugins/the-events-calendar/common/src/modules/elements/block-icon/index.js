/**
 * External dependencies
 */
import React from 'react';

/**
 * Internal dependencies
 */
import { TEC } from '@moderntribe/common/icons';
import './style.pcss';

export default () => (
	<div className="tribe-editor__icons__container tribe-editor__icons--tec">
		<TEC />
	</div>
);
