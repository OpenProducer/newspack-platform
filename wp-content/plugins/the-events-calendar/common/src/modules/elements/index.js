export { default as Accordion } from './accordion/element';
export { default as BlockIcon } from '@moderntribe/common/elements/block-icon';
export { default as Button } from './button/element';

export { default as Counter } from './counter/element';
export { default as Image } from './image/element';
export { default as ImageUpload } from './image-upload/element';
export { default as LabeledItem } from './labeled-item/element';
export { default as LabelWithLink } from './label-with-link/element';
export { default as LabelWithModal } from './label-with-modal/element';
export { default as Link } from './link/element';
export { default as ModalButton } from './modal-button/element';
export { default as TimePicker } from './time-picker/element';
export { default as Tooltip } from './tooltip/element';
export { default as DayPickerInput } from './day-picker-input/element';
export { default as CreatableSelect } from './creatable-select/element';
export { default as Placeholder } from './placeholder/element';
export { default as Heading } from './heading/element';
export { default as Paragraph } from './paragraph/element';

// Inputs
export { default as Input } from './input/element';
export { default as UrlInput } from './url-input/element';
export { default as NumberInput } from './number-input/element';
export { default as Radio } from './radio/element';
export { default as RadioInput } from './radio-input/element';
export { default as Checkbox } from './checkbox/element';
export { default as CheckboxInput } from './checkbox-input/element';
export { default as Select } from './select/element';
export { default as Textarea } from './textarea/element';

/**
 * @todo move this into Editor Module
 */

import './style.pcss'
