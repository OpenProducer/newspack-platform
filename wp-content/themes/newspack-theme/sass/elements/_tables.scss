table {
	margin: 0 0 $size__spacing-unit;
	border-collapse: collapse;
	width: 100%;
	font-size: $font__size-sm;

	td,
	th {
		padding: 0.5em;
		border: 1px solid $color__text-light;
		word-break: break-all;
	}
}
