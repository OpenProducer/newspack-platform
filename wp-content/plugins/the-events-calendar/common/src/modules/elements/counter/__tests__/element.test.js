/**
 * External dependencies
 */
import React from 'react';
import renderer from 'react-test-renderer';

/**
 * Internal dependencies
 */
import { Counter } from '@moderntribe/common/elements';

describe( 'Counter Element', () => {
	it( 'renders counter', () => {
		const component = renderer.create( <Counter /> );
		expect( component.toJSON() ).toMatchSnapshot();
	} );

	it( 'renders counter with class', () => {
		const component = renderer.create( <Counter className="test-class" /> );
		expect( component.toJSON() ).toMatchSnapshot();
	} );

	it( 'renders counter with count', () => {
		const component = renderer.create( <Counter count={ 42 } /> );
		expect( component.toJSON() ).toMatchSnapshot();
	} );

	it( 'renders counter with label', () => {
		const component = renderer.create( <Counter label="test-label" /> );
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
