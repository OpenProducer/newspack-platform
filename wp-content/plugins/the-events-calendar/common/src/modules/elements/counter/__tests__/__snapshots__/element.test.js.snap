// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter Element renders counter 1`] = `
<div
  className="tribe-editor__counter"
>
  <span
    className="tribe-editor__counter__count"
  />
  <span
    className="tribe-editor__counter__label"
  />
</div>
`;

exports[`Counter Element renders counter with class 1`] = `
<div
  className="tribe-editor__counter test-class"
>
  <span
    className="tribe-editor__counter__count"
  />
  <span
    className="tribe-editor__counter__label"
  />
</div>
`;

exports[`Counter Element renders counter with count 1`] = `
<div
  className="tribe-editor__counter"
>
  <span
    className="tribe-editor__counter__count"
  >
    42
  </span>
  <span
    className="tribe-editor__counter__label"
  />
</div>
`;

exports[`Counter Element renders counter with label 1`] = `
<div
  className="tribe-editor__counter"
>
  <span
    className="tribe-editor__counter__count"
  />
  <span
    className="tribe-editor__counter__label"
  >
    test-label
  </span>
</div>
`;
